xtask/bench/benches/graphql_formatter.rs

```diff
@@ -0,0 +1,40 @@
+use std::collections::HashMap;
+use xtask_bench::{bench_formatter_group, TestCase};
+use xtask_bench::{criterion_group, criterion_main, Criterion};
+#[cfg(target_os = "windows")]
+#[global_allocator]
+static GLOBAL: mimalloc::MiMalloc = mimalloc::MiMalloc;
+
+#[cfg(all(
+    any(target_os = "macos", target_os = "linux"),
+    not(target_env = "musl"),
+))]
+#[global_allocator]
+static GLOBAL: tikv_jemallocator::Jemalloc = tikv_jemallocator::Jemalloc;
+
+// Jemallocator does not work on aarch64 with musl, so we'll use the system allocator instead
+#[cfg(all(target_env = "musl", target_os = "linux", target_arch = "aarch64"))]
+#[global_allocator]
+static GLOBAL: std::alloc::System = std::alloc::System;
+fn bench_graphql_formatter(criterion: &mut Criterion) {
+    let mut all_suites = HashMap::new();
+    all_suites.insert("graphql", include_str!("libs-graphql.txt"));
```

Comment:
Currently, we have an issue. Files are stored and saved using their name, but the graphql files have the same name, so only the first one of the list is benched